internal/jimm/controller_test.go

```diff
@@ -1471,6 +1472,12 @@ func TestInitiateMigration(t *testing.T) {
 	}
 }
 
+// newVersion wraps version.MustParse to return a *version.Number
+func newVersion(s string) *version.Number {
```

Comment:
This test function was previously inside model_test.go so I've copied it here into controller_test.go